### PR TITLE
Add simplistic python support

### PR DESCRIPTION
**Pull request by nlopes**
This doesn't (yet) understand requirements files with includes and repository entries.
For some simple cases this should be a good start.

### CODE REVIEW

**Comment by greysteil** [comment]
:heart:

File: app/dependency_file_parsers/python.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    def initialize(dependency_files:)
+      @requirements = dependency_files.find { |f| f.name == "requirements.txt" }
+      raise "No requirements.txt!" unless @requirements
+    end
+
+    def parse
+      PythonHelpers.requirements_parse(@requirements.content).
+        each_with_object([]) do |pkg, deps|
+        deps << Dependency.new(name: pkg[0],
+                               version: pkg[1].match(/[\d\.]+/).to_s)
```
Comment (hmac) [inline app/dependency_file_parsers/python.rb]:
```ruby
.map do |package|
  Dependency.new(name: package[0], version: package[1].match(...))
end
```

File: app/dependency_source_code_finders/python.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+require "./app/dependency_source_code_finders/base"
+
+module DependencySourceCodeFinders
+  class Python < Base
+    private
+
+    def look_up_github_repo
+      @github_repo_lookup_attempted = true
+      pypi_url = URI("https://pypi.python.org/pypi/#{dependency_name}/json")
+      package = JSON.parse(Net::HTTP.get(pypi_url))
```
Comment (hmac) [inline app/dependency_source_code_finders/python.rb]: We'll probably want to handle request failures here (timeouts etc).

File: lib/python_helpers.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+      next if line.nil?
+      next unless line.match(SEPARATOR_MATCH)
+      dependencies << PythonHelpers.parse_line(line)
+    end
+    dependencies
+  end
+
+  def self.parse_line(line)
+    name, _, version = line.split(SEPARATOR_MATCH)
+    [name, version]
```
Comment (hmac) [inline lib/python_helpers.rb]: scrap that, probs overkill

File: app/dependency_source_code_finders/python.rb (hunk 2 above)
Comment (nlopes) [inline app/dependency_source_code_finders/python.rb]: Decided to leave for now.

**Pull request merged**